code-challenges.test.js

```diff
@@ -23,13 +23,26 @@ var colors1 = ["purple", "blue", "green", "yellow", "pink"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
-
-
+describe ("when 'removeShuffle' is called", () => {
+    let colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    let expect1 = ["yellow", "blue", "pink", "green"]
+    let colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+    let expect2 = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+    it ("removes first item and shuffles from 'colors1'", () => {
+        expect (removeShuffle(colors1).sort()).toEqual(expect1.sort())
```

Comment:
this is good logic for testing the randomizing logic. Also it's great that you created another expected array for the expected outcome because the test will actually be comparing two separate arrays (not referencing the same array). AWESOME! 💯